.bin/esy.js

```diff
@@ -1,170 +1,13 @@
 #!/usr/bin/env node
-// @flow
 
-let Module = require('module');
-let childProcess = require('child_process');
-let removeTypes = require('flow-remove-types');
+const fs = require('fs');
+const path = require('path');
 
-let super_compile = Module.prototype._compile;
-Module.prototype._compile = function _compile(source, filename) {
-  let transformedSource = removeTypes(source)
-  super_compile.call(this, transformedSource, filename);
-};
+const DEV = process.env.ESY_DEV || !fs.existsSync(path.join(__dirname, '..', 'lib'));
```

Comment:
If `__dirname/../lib` is not availabe we use code from `src/`, another way to force `src/` using `ESY_DEV` env var.